### PR TITLE
Update to psr/simple-cache v3.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

**Comment by othercorey** [comment]
Did they seriously do another 2 major release?

**Comment by ADmad** [comment]
> Did they seriously do another 2 major release?

After months long saga :)

File: composer.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
         "psr/http-server-handler": "^1.0",
         "psr/http-server-middleware": "^1.0",
         "psr/log": "^3.0",
-        "psr/simple-cache": "^1.0.0"
+        "psr/simple-cache": "^3.0"
```
Comment (othercorey) [inline composer.json]: We can probably support 2 || 3 if 2 is the parameter changes.

Comment (ADmad) [reply composer.json]: Done

**Pull request merged**